analyzers/ant/ant_test.go

```diff
@@ -0,0 +1,34 @@
+package ant_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/fossas/fossa-cli/analyzers/ant"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+func TestDiscoveryRoot(t *testing.T) {
+	modules, err := ant.Discover("testdata/valid", make(map[string]interface{}))
+	assert.NoError(t, err)
+	assert.Equal(t, 1, len(modules))
+	assert.Contains(t, modules, module.Module{
+		Name:        "valid",
+		Type:        pkg.Ant,
+		BuildTarget: "testdata/valid",
+		Dir:         "testdata/valid",
+	})
+}
+func TestDiscoveryEmptyRoot(t *testing.T) {
+	modules, err := ant.Discover("testdata", make(map[string]interface{}))
+	assert.NoError(t, err)
+	assert.Equal(t, 1, len(modules))
+	assert.Contains(t, modules, module.Module{
+		Name:        "valid",
+		Type:        pkg.Ant,
+		BuildTarget: "testdata/valid",
+		Dir:         "testdata/valid",
+	})
+}
```

Comment:
You probably also want a test for making sure a project _without_ a `lib` does not add an extra module.

Reply:
This is what the second test case is doing but I will make that more clear.